tests/merged/test_helpers.py

```diff
@@ -225,28 +231,52 @@ def test_apply_subtractive_rule() -> None:
                 "identifier": Identifier.generate(seed=42),
             },
         ),
-        ([], None, "One of rule_set or extracted_items is required."),
+        ([], None, True, "One of rule_set or extracted_items is required."),
+        (
+            [
+                ExtractedActivity(
+                    title=Text(value="Burger flipping"),
+                    contact=Identifier.generate(seed=97),
+                    responsibleUnit=Identifier.generate(seed=98),
+                    identifierInPrimarySource="BF",
+                    hadPrimarySource=Identifier.generate(seed=99),
+                ),
+            ],
+            ActivityRuleSetRequest(
+                subtractive=SubtractiveActivity(title=Text(value="Burger flipping")),
+            ),
+            False,
+            {
+                "contact": [Identifier.generate(seed=97)],
+                "identifier": Identifier.generate(seed=42),
+                "responsibleUnit": [Identifier.generate(seed=98)],
+            },
+        ),
     ],
     ids=(
-        "extracted_items_and_rule_set",
-        "only_rule_set",
-        "only_extracted_items",
+        "extracted items and rule set",
```

Comment:
These cases do neither check the type of the returned item (merged or preview) nor test if validation fails if the `validate_cardinality` is set and the provided params lead to a broken merged item. Could you expand the test cases?

Reply:
> These cases do neither check the type of the returned item 

fixed 1f2091f

Reply:
> Could you expand the test cases?

see cf1c39d